CardScan/Classes/SsdDetect.swift

```diff
@@ -58,15 +56,21 @@ public struct SsdDetect {
         
     }
     
-    public static func initializeModels( contentsOf url: URL) throws {
-        if SsdDetect.ssdModel == nil {
-            SsdDetect.ssdModel = try SSD(contentsOf: url)
-            }
+    public init(){
         if SsdDetect.priors == nil{
             SsdDetect.priors = PriorsGen.combinePriors()
         }
+        
+    }
+    
+    public static func initializeModels( contentsOf url: URL) throws {
+        if SsdDetect.ssdModel == nil {
+            SsdDetect.ssdModel = try SSD(contentsOf: url)
```

Comment:
convert to try? and add a guard to predict

Reply:
Made the requested changes.